src/input.jl

```diff
@@ -0,0 +1,96 @@
+
+struct GenericLinearObservation <: AbstractObservation
+end
+struct GenericLinearEvolution{TPhi,Tb,TQ} <: AbstractEvolution
```

Comment:
Is there a need for this type to be parametric? `GenericLinearObservation` is parameter-less.

Reply:
Good catch.